module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -327,6 +338,9 @@ public function init()
         $this->sortItemsBySerialIssue
             = $this->config['Holdings']['sortBySerialIssue'] ?? true;
 
+        $this->locationConfig
+        = strtolower(trim($this->config['ItemTypeRenewalBlockMappings']['Location'] ?? 'branch'));
```

Comment:
This line should be indented, and the section name isn't quite right. :)

Please make the setting start with lower-case letter. And perhaps make it `locationField` to be a bit more specific?

I just realized that the example KohaRest.ini doesn't actually contain the Holdings section (oops!). Sorry about the confusion this may have caused. I'll put it on my TODO list to augment it later.

Reply:
thanks @EreMaijala ! I've added those changes in my latest follow-up. 

I'm not sure if my addition of [Holdings] to KohaRest.ini is enough?